File: src/statesman/commands/ReturnCommand.java
package statesman.commands;

public class ReturnCommand extends JumpCommand {

    public static final String ID = "ret";

    public ReturnCommand() {
        super();
    }

    @Override
    public Command createInstance(String[] arguments) {
        return new ReturnCommand();
    }

    @Override
    public void execute() {
    }

    public int getJumpIndex() {
        return Integer.MAX_VALUE;
    }

}

File: src/Commands/ReturnCommand.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Statesman.Commands
{
    public class ReturnCommand : JumpCommand
    {
        public static readonly string ID = "ret";

        public ReturnCommand()
            : base()
        {
        }

        public override Command createInstance(string[] arguments)
        {
            return new ReturnCommand();
        }

        public override int getJumpIndex()
        {
            return int.MaxValue;
        }
    }
}
